ps/jenkins/percona-server-for-mysql-8.0.groovy

```diff
@@ -207,11 +210,124 @@ def package_tests_ps80(def nodes) {
     }
     parallel stepsForParallel
 }
+def docker_test() {
+    def stepsForParallel = [:] 
+        stepsForParallel['Run for ARM64'] = {
+            node('docker-32gb-aarch64') {
+                stage("Docker image for ARM64") {
```

Comment:
name this stage where we confirm docker image version 

something like "docker image version check for arm" 

Same for all relevant stages related to it